### DISCUSSION TITLE
Dependency graph: show the primary leading to vulnerable package

### DISCUSSION THREAD

**Discussion by ksummerlin** [community/community#29855] [Code Security]
Is there a way for the Dependency graph to show the primary dependency that leads to a sub-dependency alert? For instance:

![image](https://user-images.githubusercontent.com/104603/185237412-c57d0975-a9e7-42d3-a8b1-8a0f0b159016.png)

All 4 of these alerts are generated by sub-dependency of another package that is the primary. But it is really difficult to narrow down which primary dependency includes these packages.

**Comment by Wazbat** [comment]
I'd like to see this feature too. I know the `package.json` section shows sub-dependencies, but it doesn't show which have vulnerabilities

For now what you can do, if it's not too many, is scroll down to and expand all of the dependencies in the `package.json` section, then `ctrl + F` and search for one of the vulnerable `package-lock.json` dependencies

Though the above sadly does not work if the vulnerable package is another layer deep. You just have to keep searching frustratingly
